src/GraphQL/Schemas/Primary/Mutations/AddToTrackedMutation.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Enjin\Platform\GraphQL\Schemas\Primary\Mutations;
+
+use Closure;
+use Enjin\Platform\Enums\Global\ModelType;
+use Enjin\Platform\GraphQL\Schemas\Primary\Traits\InPrimarySchema;
+use Enjin\Platform\Interfaces\PlatformGraphQlMutation;
+use Enjin\Platform\Models\Syncable;
+use Enjin\Platform\Rules\MinBigInt;
+use Enjin\Platform\Services\Database\CollectionService;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Support\Mutation;
+
+class AddToTrackedMutation extends Mutation implements PlatformGraphQlMutation
```

Comment:
User's will be able to see what's has been tracked in the UI because the data will show in the various tables so I don't think a specific query for tracked data is needed, nor do I think we would need to scope it.  What could be helpful though is some kind of UI component to see whether a collection (or other syncable in future) will be synced in future (since data isn't removed if later untracked), this could probably be added to the various model types as an additional field.